app/migrations/20241003214418-create_assistant_messages_table.cjs

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const sql_up = `create type role_enum as enum('user', 'assistant');
```

Comment:
You could use a transaction here to make sure either all of these SQL commands are executed or none are. Usually advisable once you have to do multiple queries in one migration.

Reply:
Done